src/reducers/index.js

```diff
@@ -80,7 +80,9 @@ function getInitialState() {
       fixedColumnsCount: 0,
       fixedRightColumnsCount: 0,
       scrollableColumnsCount: 0,
-      getColumnGroup: () => {},
+      getFixedColumnGroup: () => {},
+      getFixedRightColumnGroup: () => {},
+      getScrollableColumnGroup: () => {},
```

Comment:
Column group API change to make it consistent with the existing column API.

I grabbed this along the way because this made more sense and also because it simplified writing the public API methods.